tests/techmap/zinit.ys

```diff
@@ -0,0 +1,23 @@
+read_verilog -icells <<EOT
```

Comment:
This test would be much more useful if it actually verified the init=0 case as well.